doc/content/modules/installation-guide/pages/how-tos/install.adoc

```diff
@@ -3,17 +3,41 @@
 This section delves into the detailed step-by-step instructions for installing {product}.
 Detailed explanations walk you through each step.
 
+== Full docker installation of {product}
+
 [INFO]
 ====
 Before you proceed with the installation, please ensure you have the following xref:requirements.adoc[prerequisites]:
 
-* xref:requirements.adoc[Java supported version]
-* https://www.docker.com/[Docker] installed, or an existing xref:requirements.adoc[supported database] installation with a user that has administrator privileges.
+* https://www.docker.com/[Docker] installed.
 ====
 
-== Get {product}
+{product} is distributed as a docker container.
+To install {product}, just download this https://github.com/eclipse-syson/syson/blob/{syson_tag}/docker-compose.yml[docker-compose] file and run the following docker command near the _docker-compose.yml_ file you've just downloaded:
+[source, bash]
+----
+docker compose up
+----
+
+It will install {product} (in the version corresponding to the tagged version of the documentation you are reading) and the postgres database in a composed container. 
+
+If you want to install a specific version, just edit the _docker-compose.yml_ and replace the tagged version by the one you want.
+
+You can also change the version of the documentation (use the menu on the bottom left corner) and download the _docker-compose.yml_ of this version.
```

Comment:
This will not change the docker compose file , I do not understand why you put that here?

Reply:
Yes it will, because the _docker-compose.yml_ will change on each release to contain the appropriate version of SysON.

Reply:
But I can remove the sentence if you think that it will confuse the users.